application/helpers/expressions/em_manager_helper.php

```diff
@@ -9800,6 +9752,81 @@ public static function htmlSpecialCharsUserValue($string)
         $string = str_replace(["{","}"], ["&#123;","&#125;"], $string);
         return $string;
     }
+
+    /**
+     * Save the starting values to the database
```

Comment:
Missing @return boolean|void, but can be only @return void, right?

Reply:
Why return `false` if the return value is never checked, tho?

Reply:
Well, you never know if that piece of data will be usefull to someone in the future
I would say is a habit

Reply:
> Well, you never know if that piece of data will be usefull to someone in the future
> I would say is a habit

Yes, like it :) but not necessary …